src/main/java/org/apache/commons/lang3/time/DurationFormatUtils.java

```diff
@@ -190,9 +190,6 @@ public static String formatDurationWords(
                     duration = tmp;
                     tmp = StringUtils.replaceOnce(duration, " 0 minutes", StringUtils.EMPTY);
                     duration = tmp;
-                    if (tmp.length() != duration.length()) {
-                        duration = StringUtils.replaceOnce(tmp, " 0 seconds", StringUtils.EMPTY);
-                    }
```

Comment:
Or is it a bug? Maybe a copypasta bug where accidentally lines 190 and 192 were duplicated? Because there are `replaceOne` for "hour" and "minute", and this block that was removed has the part for "seconds".

Reply:
Hi @kinow 
Yes, could be a bug as well.  In any case we don't need that conditions.
TY